test/util/helpers.ts

```diff
@@ -145,7 +145,6 @@ export function buildStubForBlockNumberRequest(
   return {
     request: {
       method: 'eth_blockNumber',
-      params: [],
```

Comment:
Hmm, with v9.0.2 we should be able to restore this. Though I'm not sure whether it matters

Reply:
Good call. It _looks _like it shouldn't matter but I'm better safe than sorry here so putting it back here as well.

Reply:
At least it's good to see it work fine with both variants after `9.0.2`